ignite/metrics/loss.py

```diff
@@ -1,4 +1,7 @@
 from __future__ import division
+from typing import Callable, Union, Optional, Sequence
```

Comment:
Maybe we can remove above line too as no more python 2:
```python
from __future__ import division
```

Reply:
Will do this in a separate PR